collector_module/opmon_mongodb_maintenance/create_indexes.py

```diff
@@ -39,6 +39,7 @@
         'indexes': [
             [('xRequestId', ASC)],
             [('correctorTime', ASC)],
+            [('insertTime', ASC)],
             [('correctorStatus', ASC), ('client.requestInTs', ASC)],
```

Comment:
But shoudln’t it be a separate migration file? Or these scripts are rerun everytime?

Reply:
This script is run every time a new MongoDB user is created. There are no any migration files here.